payjoin/src/send/mod.rs

```diff
@@ -805,6 +818,9 @@ fn find_change_index(
     Ok(Some((check_fee_output_amount(output, fee, clamp_fee_contribution)?, index)))
 }
 
+/// Checks that the change output index is valid
+/// and that the amount is not less than the fee we are suppose to
+/// be contributing.
```

Comment:
```suggestion
/// Check that the change output index is not out of bounds
/// and that the additional fee contribution is not less than specified.
```

- Use imperative mood.
- Saying "and that the additional fee contribution is not less than specified" covers both amount and clamp parameters.

The "and..." suggests to me the name is incorrect (should be check_change) but that's a nit and suggests the two *_change_index calls in determine_fee_contribution could both be named *_change instead. But this is a major nit and does not actually need to change.